core/ui/compose/common/src/test/kotlin/app/k9mail/core/ui/compose/common/window/WindowSizeClassTest.kt

```diff
@@ -0,0 +1,80 @@
+package app.k9mail.core.ui.compose.common.window
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import org.junit.Test
+
+class WindowSizeClassTest {
+
+    @Test
+    fun `should return compact when width is less than 600`() {
+        // given
+        val width = 599
+
+        // when
+        val windowSizeClass = WindowSizeClass.fromWidth(width)
+
+        // then
+        assertThat(windowSizeClass).isEqualTo(WindowSizeClass.Compact)
+    }
+
```

Comment:
The lower bound check for `WindowSizeClass.Medium` is missing, i.e. `should return medium when width is 600`.